### PR TITLE
Show a specific error for hs_disabled

### PR DESCRIPTION
**Pull request by Half-Shot**
We occasionally get support queries because the current error is not specific enough (people tend to think they have an MAU issue).

Having this error would at least make support handling a bit easier as the user is likely to report the specific error message rather than the generic one.

### CODE REVIEW

**Comment by Twi1ightSparkle** [comment]
On the behalf of the support team, yes please!

File: src/components/structures/LoggedInView.tsx [hunk 1]
```diff
@@ -94,7 +94,7 @@ interface IProps {
 
 interface IUsageLimit {
     // eslint-disable-next-line camelcase
-    limit_type: "monthly_active_user" | string;
+    limit_type: "monthly_active_user" | "hs_disabled" | string;
```
Comment (t3chguy) [inline src/components/structures/LoggedInView.tsx:97]: Where is this specced? Given that its not a namespaced value it should be somewhere :(

https://matrix.org/docs/spec/client_server/r0.6.1#id165

**Comment by Half-Shot** [comment]
Both failing tests failed with non-code related reasons, have rerun them.

File: src/components/structures/LoggedInView.tsx (hunk 1 above)
Comment (Half-Shot) [reply src/components/structures/LoggedInView.tsx:97]: Good question. it really shouldn't have ended up in synapse without a namespace.

Comment (Half-Shot) [reply src/components/structures/LoggedInView.tsx:97]: Coming back to this, `limit_type`is not specced but used in synapse as per https://github.com/matrix-org/synapse/issues/9237.

I'd like to merge this to fix usability from a pragmatism perspective, on the basis that this is just an extra mapping of a string enum to some human readable text and is unlikely to result in unusual behaviour

Comment (t3chguy) [reply src/components/structures/LoggedInView.tsx:97]: Can you stick a comment above this line with bit of text + issue link please

Comment (Half-Shot) [reply src/components/structures/LoggedInView.tsx:99]: Added

**Review by t3chguy — APPROVED** [review]
Code LGTM, tests might need latest develop pulled in

**Pull request merged**